ui/components/connections/index.js

```diff
@@ -349,9 +349,7 @@ function Connections(props) {
     ['environments', 'm'],
     ['kind', 'm'],
     ['type', 's'],
-    ['sub_type', 'm'],
-    ['updated_at', 'l'],
-    ['created_at', 'na'],
```

Comment:
@sudhanshutech Created At could be quite helpful... it should probably be included.

Reply:
ok , i removed because we were already showing date in row expansion so thought need not to be in two places. In that case i will re add it

Reply:
@sudhanshutech Not having it means that users can sort based on date, though. 

A compromise is to have the column included, but not displayed by default.

Reply:
ok yes, i added it back it back